stacker/tests/test_builder.py

```diff
@@ -84,3 +90,21 @@ def test_stack_params_dont_override_given_params(self):
         required = ['Address']
         result = handle_missing_parameters(def_params, required, stack)
         self.assertEqual(result, def_params.items())
+
+
+class TestBuilder(unittest.TestCase):
+
+    def setUp(self):
+        self.builder = Builder('us-east-1', 'namespace')
+
+    def test_resolve_parameters_referencing_non_existant_output(self):
```

Comment:
Is there maybe any refactoring we could do to make testing resolve_parameters better?  I had planned to dive back into that sometime soon - this is fine for now, but if you want to take a stab at that that'd be awesome.